ext/native-decls/GetHeliRearRotorHealth.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: CFX
+apiset: server
+game: gta5
+---
+## GET_HELI_REAR_ROTOR_HEALTH
+
+```c
+float GET_HELI_REAR_ROTOR_HEALTH(Vehicle vehicle);
+```
+
+**Note**: This native is a getter for [GET_HELI_TAIL_ROTOR_HEALTH](https://docs.fivem.net/natives/?_0xAE8CE82A4219AC8C)
+
+## Parameters
+* **vehicle**: The target vehicle.
+
+## Return value
+Returns the health of the helicopter's rear rotor. The maximum health value is `1000`.
```

Comment:
Has it been verified that this actually clamps the max health?

Reply:
Yes, but I may be wrong, feel free to double-check